### ISSUE TITLE
Clarify DOI procedure for ACL-supported SIGs

### ISSUE THREAD

**Issue by mdingemanse** [acl-org/acl-anthology#2836]
The [ACL FAQ](https://aclanthology.org/faq/doi/) recommends DOIs and notes that "the DOI information above is only applicable to ACL sponsored events by ACL, its chapters or SIGs". Indeed for many ACL SIGs the papers duly get DOIs, and have gotten them for years. 

However, there seems to be a bunch of SIGs for which the process somehow got broken in recent years — e.g., [SIGDial papers](https://aclanthology.org/sigs/sigdial/) had DOIs until [at least 2019](https://aclanthology.org/volumes/W19-59/) but [recent years don't](https://aclanthology.org/volumes/2022.sigdial-1/); and likewise INGL had DOIs in [2019](https://aclanthology.org/volumes/W19-86/) but [not now](https://aclanthology.org/volumes/2023.inlg-main/)). Both of these are connected to ACL Special Interest Groups and so should in my reading of the above policy qualify for getting DOIs.

When asking the SIGdial and INLG publication chairs about this there seems to be a lack of institutional knowledge about the process of getting DOIs assigned. Indeed searching on my own in the support and FAQ materials it is not clear there is a procedure in place. (Though scripts exist for batch DOI assignment, it is not clear who should run them and when.) I'm guessing that at some point, a bit of the institutional knowledge about the process fell by the wayside and now at least some publication teams of some SIGs people don't generally know how to get DOIs. 

If any of this makes sense, and if ACL stands by the recommendation to use DOIs and by the possibility of assigning them for its SIG venues (all of which I hope — DOIs are good for citability, findability, durability), then my suggestion would be to clarify this process as part of the SIG venue publication procedure. If it is already described, I would be very happy to have a URL that we can point relevant publication chairs to.

**Comment by dmhowcroft** [comment]
Just commenting to formally request DOIs for SIGGEN for our events from 2020-2023. Confirmed that the main conf proceedings are missing DOIs:

- https://aclanthology.org/volumes/2020.inlg-1/
- https://aclanthology.org/volumes/2021.inlg-1/
- https://aclanthology.org/volumes/2022.inlg-main/
- https://aclanthology.org/volumes/2023.inlg-main/

I recall when I was pub chair a couple of years ago, ACLPUB had special instructions about how to get the ISBNs for the proceedings, but didn't say anything about DOIs, so I had assumed it was a semi-automated, Anthology-internal process that happened automatically after ingestion.

**Comment by mjpost** [comment]
It is semi-automated, but sometimes it gets overlooked. @anthology-assist can you add these please?

**Comment by mdingemanse** [comment]
Can I then formally request that in addition to the siggen years listed above by @dmhowcroft , dois also be added to the same years of SIGdial:

https://aclanthology.org/volumes/2020.sigdial-1/

https://aclanthology.org/volumes/2021.sigdial-1/

https://aclanthology.org/volumes/2022.sigdial-1/

And 2023: https://aclanthology.org/events/sigdial-2023/#2023sigdial-1

Thank you for your assistance!

**Comment by mdingemanse** [comment]
@mjpost writes: 
>It is semi-automated, but sometimes it gets overlooked.
I am glad to learn that there wasn't a change of procedure, just some institutional knowledge being lost somewhere. My suggestion in OP was to clarify this process as part of the publication procedure. I would be happy to do a PR but rummaging around in various places I don't really see a central guide for publication chairs (which would have helped the SIGDial chair I first took this up with, who didn't seem to know about DOIs being a separate piece of infrastructure) and I am not familiar enough with the relevant processes to suggest exactly what risks getting overlooked where.

**Comment by dmhowcroft** [comment]
The central guide for pubchairs is on the ACLPUB website/repo: https://acl-org.github.io/ACLPUB/advice.html

However, these instructions don't mention DOIs. We should probably open a separate Issue or PR on the ACLPUB repo about the instructions, I would guess?

**Comment by mdingemanse** [comment]
The [last edits](https://github.com/acl-org/ACLPUB/tree/master/doc/pubchair) to most of those docs are >12 years ago. Something that looks like a DOI encouragement policy is expressed [here](https://acl-org.github.io/ACLPUB/formatting.html):

> As part of our work to make ACL materials more widely used and cited outside of our discipline, ACL has registered as a CrossRef member, as a registrant of Digital Object Identifiers (DOIs), the standard for registering permanent URNs for referencing scholarly materials.

But no specific guidance on DOIs appears to be available for publication chairs. Before suggesting some I'll await word (from @mjpost @anthology-assist ?) on whether this is indeed the most up to date info publication chairs are getting, and what their preferred method would be for ensuring SIG-contributed proceedings get ACL-assigned DOIs.

**Comment by mdingemanse** [comment]
Sorry to bump this @mjpost and @anthology-assist but I noticed the SIGGEN and SIGDIAL 2023 proceedings have since been published (and an "INGL cleanup" [issue](https://github.com/acl-org/acl-anthology/issues/2823) in which I referred to this as an open request was since closed) — but none of the papers have been assigned DOIs yet. 

This current issue is not perspicuously titled so I can imagine it's easy to miss. Would it help if we branched out into a specific issue to explicitly make/repeat the requests for the relevant SIGs and years? Or can we assume that since it's been marked "triaged" somebody will get to it eventually? 

I am sorry to be nagging here but it seems ACL agrees that DOIs are a pretty important piece of scientific infrastructure and it would be ideal if there is not too much time between a paper appearing and it getting a DOI.
